lib/rubocop/cop/rspec/mixin/top_level_group.rb

```diff
@@ -7,6 +7,11 @@ module RSpec
       module TopLevelGroup
         extend RuboCop::NodePattern::Macros
 
+        DEPRECATED_PRIVATE_METHOD_WARNING =
+          'top_level_group? is deprecated and will be ' \
+          'removed in the next major version of Rubocop/RSpec.' \
+          'Please copy the method if you need it for your mixin.'
```

Comment:
Is it a good advice to copy?
We’ve stopped using it for a reason?
Was that performance? Should we say to consider using the hook instead?

Reply:
I actually don't know what the hook is. Do you want to suggest language?

Reply:
I removed the copying advice